app/multitenant/dynamo_collector.go

```diff
@@ -78,6 +81,24 @@ var (
 		Name:      "nats_requests",
 		Help:      "Number of NATS requests.",
 	}, []string{"method", "status_code"})
+
+	memcacheHits = prometheus.NewCounter(prometheus.CounterOpts{
+		Namespace: "scope",
+		Name:      "memcache_hits",
+		Help:      "Reports that missed our in-memory cache but went to our memcache",
+	})
+
+	memcacheMiss = prometheus.NewCounter(prometheus.CounterOpts{
+		Namespace: "scope",
+		Name:      "memcache_miss",
+		Help:      "Reports that missed both our in-memory cache and our memcache",
+	})
```

Comment:
A better way of doing this is to have a single memcache counter, with a label for hit or miss.  Makes writing the queries easier.  You could also fix up the other hit/miss counter too.